src/tests/miscellaneous/mock-adb/app/bin.js

```diff
@@ -48,7 +48,15 @@ async function main() {
     const configContent = fs.readFileSync(configPath);
     const config = JSON.parse(configContent);
 
-    const outputFile = path.join(path.dirname(process.execPath), 'mock_adb_output.json');
+    const currentContext = fs.readFileSync(
+        path.join(runtimeDirname, 'latestAdbContext.txt'),
```

Comment:
Is there an easy to have 'latestAdbContext.txt' defined in a single location, instead of defining it independently in 2 different files?